bytes.podspec

```diff
@@ -20,11 +20,11 @@ Pod::Spec.new do |s|
 
   s.license      = "MIT"
 
-  s.author      = { "Cornelius Horstmann" => "horstmann@tbointeractive.com", "Thorsten Stark" => "stark@tbointeractive.com", "Bernhard Eiling" => "eiling@tbointeractive.com", "Pascal Stüdlein" => "stuedlein@tbointeractive.com"  }
+  s.author      = { "Cornelius Horstmann" => "horstmann@tbointeractive.com", "Thorsten Stark" => "stark@tbo.de", "Bernhard Eiling" => "eiling@tbointeractive.com", "Pascal Stüdlein" => "stuedlein@tbointeractive.com"  }
 
   s.platform     = :ios, "8.2"
 
-  s.source       = { :git => "https://github.com/tbointeractive/bytes.git", :tag => 'v1.0'}
+  s.source       = { :git => "https://github.com/tbointeractive/bytes.git", :tag => 'v1.1'}
```

Comment:
Just as a reminder. You need to tag it before you can publish it to Cocoapods then

Reply:
Thanks, good point!